webapp/channels/src/actions/admin_actions.jsx

```diff
@@ -91,6 +91,33 @@ export async function samlCertificateStatus(success, error) {
     }
 }
 
+export async function getIPFilters(success, error) {
```

Comment:
Ideally, you'd have the components dispatch the actions you added to `mattermost-redux` directly instead of wrapping them in this old style of Flux actions with success/error callbacks which bind to the global instance of the Redux store this way, but since they're consistent with the rest of this file file, it's fine to leave as-is.

I'm honestly surprised this file has survived for so long as-is since this is how we made all the existing Flux actions work with Redux when we first made the change to it. I guess Apps/ICU/Web Platform has always been focused more on the non-System Console app when its come to cleanup 😅

Reply:
lol it seems given 2 patterns to follow I'll happen to pick the wrong one... Every time :P 

i went this route because it seemed more correct than wrapping some anonymous function to allow for async/await in the useEffect. Thinking about it now, I guess I could have just had useEffect trigger an async function and have that perform the await. If you're ok with this way to match the rest of the file I'm cool to leave it

Reply:
Yep, that's fine by me